### ISSUE TITLE
Dashboard UI Element Thats Links To Other Dashboards | Visual Map Dashboard |  Feature Request

### ISSUE THREAD

**Issue by TahriT** [metabase/metabase#6335]
Hello Metabase Team, 
I have a feature request that could be beneficial to a lot of people in manufacturing. 

tldr;
A dashboard UI element that can link to the other dashboards. With the possibility of a color state.

**Preface:**
At my job we have a large number of machine which push data to a database. Metabase works great in handing all the data and dashboards look amazing! However there is one feature that would make a big impact on us. 

We have dashboards for each machine but need to get to each one manually through a alphabetical list, this is cumbersome when you have several machines, and just want to quickly look at one machine that is behaving different than the others. 

If we had a overview of each of the dashboards not as just a grid display but as a visual map of our floor layout it would make navigating between dashboards much easier. 

**Example:**
You click to go into dashboards but instead of the standard grid display of available dashboards you get a map that you created which links to each dashboard.  In the same way we make dashboards now by dragging in the UI element, this dashboard could have a block the represents a location or machine. Clicking on this block would bring you to that dashboard. If this block could also have a color state based on a value that would be even better for us. 

For a manufacturing setting this is gold because we need to know whats going on with each machine and to have a visual element that links with our dashboards would be extremely helpful. 

I believe you could use the existing dashboards to do this maybe and just add a way to link between dashboards with a certain block. A color state would be a great addition to that as well but maybe a separate request.
 
Thanks to the team for all the hard work!  My apologies if this was already suggested!

**Comment by flamber** [comment]
This is possible with Click Behavior: https://www.metabase.com/learn/dashboards/custom-destinations.html

As for color based on value, then perhaps either #3168 or Conditional Formatting: https://www.metabase.com/docs/latest/users-guide/05-visualizing-results.html#conditional-table-formatting
